dist/gmusic-theme.js

```diff
@@ -229,9 +229,9 @@ window.GMusicTheme = GMusicTheme;
 
 
 },{"../lib/_constants":2,"color":7}],2:[function(require,module,exports){
-module.exports = {
-  CLASS_NAMESPACE: 'gmusic-theme',
-};
+module.exports = {
```

Comment:
I always commit LF, but we've had this issue before with @chrismou 

So I just ignore it from now on 😆